sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample12_AnalyzeTextOperationAsync_AbstractiveSummarizationOperationAction.md

```diff
@@ -0,0 +1,123 @@
+# Summarize documents with abstractive summarization
+
+This sample demonstrates how to perform abstractive summarization, which can generate a summary from a text document by composing original sentences concisely and coherently to convey the most important and relevant information. To learn more about document summarization, see [here][Document_Summarization].
+
+## Create a `TextAnalysisClient`
+
+To create a new `TextAnalysisClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create with an API key.
+
+```C# Snippet:CreateTextClient
+Uri endpoint = new Uri("<your endpoint>");
+AzureKeyCredential credential = new("your apikey");
+TextAnalysisClient client = new TextAnalysisClient(endpoint, credential);;
+```
+
+The values of the `endpoint` and `apiKey` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+## Summarize one or more text documents
+
+To summarize one or more text documents using extractive summarization, call `AnalyzeTextOperationAsync` on the `TextAnalysisClient` by passing the documents as `MultiLanguageTextInput` parameter and a `AnalyzeTextOperationAction` with a `AbstractiveSummarizationOperationAction` action. This returns a `Response<AnalyzeTextOperationState>` which you can extract the `AbstractiveSummarizationOperationResult`.
```

Comment:
Do we need docs to explain the diff between abstractive and extractive summarization?

Reply:
First line gives basic explanation and links to the documentation with examples.